### DISCUSSION TITLE
Documentação para CSS

### DISCUSSION THREAD

**Discussion by dancasttro** [frontendbr/forum#57] [Perguntas e Respostas]
Olá pessoal, tudo bem?!

O que vocês usam quando precisão documentar seus CSS de uma aplicação?

**Comment by vitorbritto** [comment]
@dancasttro tomei a liberdade de abreviar o título. Ok? =]

Eu utilizo o [SassDoc](http://sassdoc.com/).

**Comment by felipefialho** [comment]
Alguém conhece uma para Stylus? c/c @willianjusten

**Comment by neysimoes** [comment]
Comecei um projeto de um cliente e to usando Codepen.

Se vc olhar os CSS externos que eu coloco tem um outro codepen de "Flat Buttons" que eu também fiz. Assim vou encadeando um codepen no outro fazendo todo a doc com exemplos.

http://codepen.io/neysimoes/pen/ZQZRLq

**Comment by dancasttro** [comment]
Blz @vitorbritto. É gostaria de algo em stylus, pois o projeto que estou usa ele

**Comment by willianjusten** [comment]
Para Sass também o utilizo o SassDoc, para stylus, eu criei um parser, que faz praticamente a mesma coisa que o SassDoc. Eu acho que nunca liberei, pq sei lá...

Vou ver se pego um dia para documentar bonitinho e libero para galera contribuir.

**Comment by felipefialho** [comment]
Faça isso @willianjusten, estamos aguardando \o

**Comment by jhonnymoreira** [comment]
@willianjusten se quiser ajudar pra isso, me dá um toque. [=
